guide: DVC Project concept page (#2754)
iterative/dvc.org: content/docs/user-guide/basic-concepts/dvc-project.md

tooltip: >-

>   Initialized by running `dvc init` in the **workspace** (typically a Git

Author: Tooltip in a tooltip? 🙂 Meta (also, doesn't work)

>   Initialized by running `dvc init` in the **workspace** (typically a Git

Reviewer: not within, but hyperlink to glossary entry at least.

  repository). It will contain the `.dvc/` directory, as well as `dvc.yaml` and
  `.dvc` files created with commands such as `dvc add` or `dvc run`.  

>   See [full concept](/doc/user-guide/basic-concepts/dvc-project).

Author: > auto-render if markdown (below) is non-empty?

Yes it was mentioned in https://github.com/iterative/dvc.org/pull/2453#pullrequestreview-711436406 (well sort of -- auto-linking from Glossary) and discussed with the team but deemed not a priority (at all). Also related to https://github.com/iterative/dvc.org/pull/2755#pullrequestreview-737852600.

>   See [full concept](/doc/user-guide/basic-concepts/dvc-project).

Author: p.s. there's an issue about this actually, I had forgotten — #2756 (will update description with details above)

p.p.s. I did apply the suggestion in this review.

>   See [full concept](/doc/user-guide/basic-concepts/dvc-project).

Author: > p.s. there's an issue about this actually, I had forgotten — #2756

p.p.p.s. never mind that's something else. Created https://github.com/iterative/dvc.org/issues/2771

---

# DVC Project

Initialized by running `dvc init` in you <abbr>workspace</abbr> directory, it
will contain all the [DVC files and directories][dvc-metafiles] including the
<abbr>cache</abbr>, `dvc.yaml` and `.dvc` files, etc. Any other files referenced
from these DVC metafiles are also considered part of the project (for example

> [metrics files](/doc/command-reference/metrics)).

Author: Thanks for the suggestion but I think both are valid since this text is right under the **DVC Project** title (so that's clearly what "it" refers to, I think). Stylistic

> [metrics files](/doc/command-reference/metrics)).

Author: Oh I see there's another copy edit that changes `"including" for long dash. Same thing though, both are valid I think. I incline to keep mine of course unless there's an error somewhere.

> Initialized by running `dvc init` in you <abbr>workspace</abbr> directory, it
> will contain all the [DVC files and directories][dvc-metafiles] including the
> <abbr>cache</abbr>, `dvc.yaml` and `.dvc` files, etc. Any other files referenced
> from these DVC metafiles are also considered part of the project (for example
> [metrics files](/doc/command-reference/metrics)).

Reviewer: ```suggestion
Initialized by running `dvc init` in your <abbr>workspace</abbr> directory, a
DVC project will contain all of the [DVC metafiles and
directories][dvc-metafiles], such as the <abbr>cache</abbr>, `dvc.yaml`, and
`.dvc` files. Any other files referenced from these DVC metafiles are also
considered part of the project (e.g. [metrics
files](/doc/command-reference/metrics)).
```

lots of typos as well. compromised.

> [metrics files](/doc/command-reference/metrics)).

Author: What typos?

[dvc-metafiles]: /doc/user-guide/project-structure

> `dvc destroy` can be used to remove all DVC-specific files from the directory,
> in effect deleting the DVC project.

## DVC repository

A DVC project in a Git repository can also be called _DVC repository_ or "the
repo". This setup enables the
[versioning features](/doc/start/data-and-model-versioning) of DVC
(recommended). Files tracked by Git are considered part of the DVC project when
referenced from DVC metafiles such as `dvc.lock`, for example source code that

> is used as a <abbr>stage</abbr> command (`cmd` field in `dvc.yaml`).

Author: Again thanks but I prefer `, for example`. It's simpler and hopefully easier to read.

> A DVC project in a Git repository can also be called _DVC repository_ or "the
> repo". This setup enables the
> [versioning features](/doc/start/data-and-model-versioning) of DVC
> (recommended). Files tracked by Git are considered part of the DVC project when
> referenced from DVC metafiles such as `dvc.lock`, for example source code that
> is used as a <abbr>stage</abbr> command (`cmd` field in `dvc.yaml`).

Reviewer: ```suggestion
A DVC project in a Git repository can also be called a _DVC repository_ or "the
repo". This setup enables the
[versioning features](/doc/start/data-and-model-versioning) of DVC
(recommended). Files tracked by Git are considered part of the DVC project when
referenced from DVC metafiles such as `dvc.lock`; for example source code that
is used as a <abbr>stage</abbr> command (`cmd` field in `dvc.yaml`).
```

tooltip: >-
  Initialized by running `dvc init` in the **workspace** (typically a Git
  repository). It will contain the `.dvc/` directory, as well as `dvc.yaml` and
  `.dvc` files created with commands such as `dvc add` or `dvc run`.  
  [More info](/doc/user-guide/basic-concepts/dvc-project)
---

# DVC Project

Initialized by running `dvc init` in your <abbr>workspace</abbr> directory, a
DVC project will contain all of the [DVC metafiles and

> directories][dvc-metafiles], such as the <abbr>cache</abbr>, `dvc.yaml`, and

Reviewer 2: cache location can be changed though

Reviewer: yes, but explicitly stating that here may seem tangential/distract from the point? I don't have strong opinions here. The `<abbr/>` tooltip will of course provide more details.

Author: > cache location can be changed"such as" leaves room for that

Author: > cache location can be changed

"such as" leaves room for that
